File: C_PROGRAMMING/C_PROGRAMS/Statements/0_Basics/program6.c

// WAP to check whether the number is even or odd, take all values from users...
//
#include<stdio.h>
void main()
{
   int num;

   printf("Enter the number: ");
   scanf("%d",&num);

   if(num%2==0){
     printf("\n %d is an even number..",num);
   }

   else{
     printf("\n %d is an odd number..",num);
    }
}

File: JAVA_PROGRAMMING/CLASS_CODES/Control_Statements/For_loop/program6.java

// WAP to check whether the given number is prime or not..


class Demo{

	public static void main(String[] args){

		int num = 3;
		int count = 0;

		for(int i = 1; i<= num; i++){

			if(num % i == 0){

				count++;
			}

		}
		if(count == 2)
			System.out.println(num +" is a Prime Number");
		else
			System.out.println(num+ " is not a Prime Number");
	}
}
